lambdas/accesstoken/src/main/java/uk/gov/di/ipv/core/accesstoken/AccessTokenHandler.java

```diff
@@ -99,6 +99,16 @@ public APIGatewayProxyResponseEvent handleRequest(
                         error.getHTTPStatusCode(), error.toJSONObject());
             }
 
+            if (authorizationCodeService.isExpired(authorizationCodeItem)) {
+                LOGGER.error(
+                        "Access Token could not be issued. The supplied authorization code has expired. Created at: {}",
+                        authorizationCodeItem.getCreationDateTime());
+                ErrorObject error =
+                        OAuth2Error.INVALID_GRANT.setDescription("Authorization code expired");
```

Comment:
We have a helper for logging OAuth errors in a structured way now. Would be good to use hear (and a bunch of other places in this class TBF). It's `LogHelper.logOauthError()`.